README.md

```diff
@@ -4,12 +4,16 @@ Generates a basic analysis of a CSV to an XLSX workbook.
 ```
 usage: csv_summary [-h] [-o OUTPUT_PATH]
                    [--category-threshold CATEGORY_THRESHOLD]
-                   [--date-format DATE_FORMAT] [-i IGNORE_VALUE]
-                   [-s NUM_SAMPLES]
-                   csv_path
+                   [--date-format DATE_FORMAT]
+                   [--date-time-format DATE_TIME_FORMAT] [-i IGNORE_VALUE]
+                   [--sheet-name SHEET_NAME] [-s NUM_SAMPLES]
+                   input
 
 positional arguments:
-  csv_path
+  input                 CSV or XSL input containing data. If XLS/XLSX, will
+                        add two extra sheets with summary/sample data. Will
+                        update in place unless -o is specified. First sheet is
+                        treated as the data unless --sheet-name is specified.
```

Comment:
A bit unclear. If you specify a sheet name, will there be four tabs? The original, one that has the new name (and has been processed to lay out columns nicely) and the two new ones?  In the case where no sheet name is used, it the original data left as is, or processed for column widths?

Reply:
I will try to clarify. The `--sheet-name` tells the script where the original raw data is (if not specified, it just uses the first sheet in the workbook). Two extra sheets are appended no matter what. Also, since the original XLS is assumed to have formatting, no changes are made to the original content - but the two extra sheets are formatted.